app/src/main/java/com/duckduckgo/app/browser/BrowserTrackersAnimatorHelper.kt

```diff
@@ -234,6 +234,7 @@ class BrowserTrackersAnimatorHelper {
             .asSequence()
             .distinct()
             .take(MAX_LOGOS_SHOWN + 1)
+            .sortedWith(compareBy { "AEIOU".contains(it.displayName[0]) })
```

Comment:
`displayName` can't be null, but could it ever be an empty string? if so, that line would crash. 

`displayName.take(1)` is safer in this regard, and we use that elsewhere in the same function so is also more consistent. Happy to switch `displayName[0]` for `displayName.take(1)` ?


Also, would be great if there were some unit tests for this comparator if it's easy to pull that out into something testable

Reply:
displayName can't be empty, we are already filtering blank names out in `Site.kt` with `.filter { it.displayName.isNotBlank() }`